test/modules/graphstorage.test.js

```diff
@@ -38,35 +38,60 @@ describe('GraphStorage module', () => {
             'host', 'port', 'max_path_length', 'database']);
         selectedDatabase.database = myDatabaseName;
 
-        systemDb = new Database();
-        systemDb.useBasicAuth('root', 'root');
-        await systemDb.createDatabase(
-            myDatabaseName,
-            [{ username: myUserName, passwd: myPassword, active: true }],
-        );
+        if (selectedDatabase.database_system === 'arango_db') {
+            systemDb = new Database();
+            systemDb.useBasicAuth('root', 'root');
+            await systemDb.createDatabase(
+                myDatabaseName,
+                [{ username: myUserName, passwd: myPassword, active: true }],
+            );
+        } else if (selectedDatabase.database_system === 'neo4j') {
+            // TODO Implement me
+        }
 
         myGraphStorage = new GraphStorage(selectedDatabase);
         expect(myGraphStorage).to.be.an.instanceof(GraphStorage);
+        myGraphStorageConnection = await myGraphStorage.connect();
+
+        myInvalidGraphStorage = new GraphStorage();
     });
 
-    it('connect() and identify()', async () => {
-        myGraphStorageConnection = await myGraphStorage.connect();
-        // TODO meaningful checks for connect()
+    beforeEach('reset ot_vertices and ot_edges collections', async () => {
+        if (selectedDatabase.database_system === 'arango_db') {
```

Comment:
same as above. 
we need to add a function, something like `clear()` in Neo4j that will clear the db and call `clear()` from the graph storage